mypy/test/testpep561.py

```diff
@@ -250,6 +277,31 @@ def test_mypy_path_is_respected(self) -> None:
                     assert returncode == 0
             finally:
                 os.chdir(old_dir)
+    
+    def test_mypy_nositepackages_setting_accepted(self) -> None:
+        with tempfile.TemporaryDirectory() as temp_dir:
+            old_dir = os.getcwd()
+            os.chdir(temp_dir)
+            try:
+                self.simple_prog.create()
+
+                mypy_config_path = os.path.join(temp_dir, 'mypy.ini')
+                with open(mypy_config_path, 'w') as mypy_file:
+                    mypy_file.write('[mypy]\n')
+                    mypy_file.write('no_site_packages=True\n')
+
+                with self.virtualenv() as venv:
+                    venv_dir, python_executable = venv
+                    self.install_package('typedpkg', python_executable)
+                    self.install_package('typedpkg-stubs', python_executable)
+                    self.simple_prog.check_mypy_run_args(
+                        python_executable,
+                        expected_out=[SimpleMsg.msg_skip_executable],
```

Comment:
The expected output is supposed to be the entire expected output. I would not use -vv and instead use the warning mypy gives for not finding the package.

Reply:
Applied! I had some trouble changing the line numbers (f.e. some had different line numbers) and had to make a new message for that. Wouldn't it be better to inject line numbers on the spot where the list `expected_out` is put together?